core/src/main/java/org/elasticsearch/common/unit/ByteSizeValue.java

```diff
@@ -107,23 +107,26 @@ public double getPbFrac() {
 
     @Override
     public String toString() {
-        long bytes = getBytes();
```

Comment:
can you revert these changes and rather create a new BytesSizeValue from the source length?